src/app/api/run/[runId]/upload/route.test.ts

```diff
@@ -55,4 +95,27 @@ describe('POST /api/run/[runId]/upload', () => {
         const response = await POST(req, { params })
         expect(response.status).toBe(400)
     })
+
+    it('should return an error if the runID has results already', async () => {
+        const mockRunId = uuidv4()
+
+        mockFs({
+            [UPLOAD_DIR]: {
+                [mockRunId]: '',
+            },
+        })
+
+        const formData = new FormData()
+        formData.append('file','')
+
+        const req = {
+            formData: async () => formData,
+        } as NextRequest
+
+        const params = { runId: mockRunId }
+
+        const response = await POST(req, { params })
+        expect(response.status).toBe(400)
+        expect((await response.json()).error).toBe('Data already exists for runId')
```

Comment:
@philschatz [had left a comment on my PR](https://github.com/safeinsights/trusted-output-app/pull/15#discussion_r1876362977) about testing for the error string of the 400 response and maybe having that not be necessary? im not sure if that applies here - i dont feel strongly either way, just pointing out for consistency

Reply:
oh yeah good point! Im happy keeping it in, doesn't matter much to me